docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
     # Use the host's IPC namespace for inter-process communication
     environment:
       - MODEL_NAME=${MODEL_NAME:-/models/gemma-2-2b-it}
+      - HF_TOKEN=${HF_TOKEN}
```

Comment:
For this we can accept it, but it is another + for using Podman etc is the easier use of secrets in the future rather than what we get with the free Docker / having to put a sidecar.